documentation/src/main/asciidoc/topics/con_xsite_backup_strategies.adoc

```diff
@@ -1,5 +1,43 @@
-[id='xsite_backup_strategies-{context}']
-= Backup Strategies
-{brandname} clusters can use different strategies for backing up data to remote sites.
+[id='backup-strategies_{context}']
+= Backup strategies
 
-{brandname} replicates across sites at the same time that writes to local caches occur. For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+{brandname} replicates data between clusters at the same time that writes to caches occur.
+For example, if a client writes "k1" to **LON**, {brandname} backs up "k1" to **NYC** at the same time.
+
+To back up data to a different cluster, {brandname} can use either a synchronous or asynchronous strategy.
+
+[discrete]
+== Synchronous strategy
+When {brandname} replicates data to backup locations, it writes to the cache on the local cluster and the cache on the remote cluster concurrently.
+With the synchronous strategy, {brandname} waits for both write operations to complete before returning.
+
+You can control how {brandname} handles writes to the local cache if backup operations fail.
+For example, you can configure {brandname} to log an exception and continue the write operation on the local cache or attempt to stop the write operation on the local cache.
```

Comment:
hmmm, still not 100% accurate. how about
"For example, you can configure {brandname} to [silently] ignore, log or throw an exception. Also, you can inject your own code to handle the backup failures.

NOTE: The local write proceeds and it is not affected by backup failure"

Reply:
thanks @pruivo I'm not sure about "NOTE: The local write proceeds and it is not affected by backup failure" - in the case of "FAIL" this isn't true, is it?

Reply:
@pruivo I get a little mixed up with the "THROW" policy: https://github.com/infinispan/infinispan/blob/36fe531f448ddba3bbbb045d393213bac5f366c7/core/src/main/resources/schema/infinispan-config-13.0.xsd#L2744 

if THROW then "attempt to stop writes to the local cache." If the bottom line is the local site has the new value does that mean Infinispan can never successfully stop writes to the local cache?

Obvs the difference between throwing an exception and logging an exception is that throwing means it tries to stop execution, whereas log just prints a message. Could the description in the xsd be wrong as well? 

Thank you.

Reply:
that is not really true (except for 2 phase commit transactions)

Reply:
ok, thanks. I've rephrased it. hopefully I'm not driving you nuts with this. :smiley_cat: